backend/src/web_server.py

```diff
@@ -4,48 +4,57 @@
 """ TODO: 
         - Change to be a DJango type web server (???) 
         - Make more scalable
-        - Add a method to turn off the server """ 
+        - !!! Add a method to turn off the server (I have no idea, tried signal handler - damon)
+        """ 
 
 import socket
 import sys
 import time
 import ast
-import threading
+import queue
 import user_commands
+
 from user_commands import userCommands
 from threading import Lock
-import datetime as datetime
+from threading import Thread
 
+import datetime as datetime
 
+# Default server values, but can be inputted by user on class initialization
 SERV_PORT       = 65432
 SERV_HOST_NAME  = '127.0.0.1'
 
 PACKET_SIZE     = 1024
-thread_print_lk = Lock()
-#REFERENCE MATERIALS
+
+# REFERENCE MATERIALS
 # https://gist.github.com/joncardasis/cc67cfb160fa61a0457d6951eff2aeae
 # https://realpython.com/python-sockets/
 
 class webServer():
 
     def __init__(self, port=SERV_PORT, hostname=SERV_HOST_NAME):
-        self.port       = port
-        self.hostname   = hostname
+        self.port           = port
+        self.hostname       = hostname
+        self.userProcesses  = {}
+        self.serverRunning  = True
+
 
     def start(self):
         """ starts the webserver up. This includes initializing the socket, 
         binding the socket for recieving, and calling the class listen function"""
+        time_now = time.strftime("%a, %d %b %Y %H:%M:%S", time.localtime())
+        print("-------SERVER STARTED!-------")
+        print("Timestamp: ", time_now)
+        print("Port: " + str(self.hostname))
+        print("Port: " + str(self.port))
+        print()
+        
+        # Initialize a server administration thread for dumplog command
+        self.initializeUserThread("admin")
```

Comment:
Initialized an "admin" thread for administrative tasks down the line, and for dumplog commands.